### ISSUE TITLE
Using Custom Grid causes Safari to choke and create whitescreen

### ISSUE THREAD

**Issue by shaneonabike** [dudaster/ele-custom-skin#412]
I discovered a strange issue whereby when I use a Custom Grid this causes Safari to hang about 50% way at loading the page. When it completes it's a completely white screen.

My custom grid isn't anything special and can recreated here:

[elementor-402-2021-01-06.json.txt](https://github.com/dudaster/ele-custom-skin/files/5777277/elementor-402-2021-01-06.json.txt)

I'm using Elementor and Elementor Pro. I went through and disabled all the plugins, but re-enabled them one by one. My issues appeared when I enabled this one, and can confirm that when I remove the Grid the page loads as expected.

**Comment by shaneonabike** [comment]
Further debugging shows me that having more than one column in the actual loop is causing this issue. Furthermore, I cannot use an Inner Column either :/. 

When I placed all the data in the main column it seems to be working just fine. I don't really understand why it's an issue but something is amuck.

**Comment by dudaster** [comment]
Safari usually has issues with image hover effects . Multiply that by 100 and it will have issues.

Please try to remove the mouse hover effects and let me know if it solves the issue.

**Comment by shaneonabike** [comment]
Actually, I don't have any image hover effects either. It's a pretty straightforward output. Here's the output for the sake of review. As I mentioned, when I remove the columns OR actually remove the grid completely it loads properly. I don't know if the DOM is too big for it to handle or what is happening. I find it really odd to be honest, but frustrating because I really really appreciated your grid system and it helped for a specific design I needed.

[elementor-370-2021-01-07.json.txt](https://github.com/dudaster/ele-custom-skin/files/5781645/elementor-370-2021-01-07.json.txt)

**Comment by dudaster** [comment]
Can you send me a screenshot of how it should look like... for me is all messed up.

**Comment by shaneonabike** [comment]
![Selection_005](https://user-images.githubusercontent.com/1126026/103900118-cf37f200-50cd-11eb-9e1d-c5754003ffaf.jpg)

For sure! You'll notice that I'm using the grid in order to actually add a grid overlay colour, which is the empty html widget. Below that in another section of two elements.

**Comment by shaneonabike** [comment]
You should be able to replicate the problem by using the basic grid I have and then adding on a loop that contains a column or inner section.

**Comment by dudaster** [comment]
if you use this loop template without the grid is loads ok on safari?

**Comment by shaneonabike** [comment]
Yes exactly!~ Which is making question why is the grid messing things up? I couldn't find any issues with Safari reported about DOM related issues... shouldn't this work?

**Comment by shaneonabike** [comment]
Were you able to reproduce this issue by any chance?

**Comment by mikriz** [comment]
I am having a very similar issue in Safari, except i am NOT using a custom grid. I simply have a 2 column layout of items, each item is made up of a dynamically added image, header and text. The only other plugins in use in this loop would be DynamicConditions. Have you had any luck solving this problem? the page loads 100% fine in other browsers. If i should open a new issue let me know. 
![Screen Shot 2021-03-02 at 9 43 23 AM](https://user-images.githubusercontent.com/21971636/109665181-e8907400-7b3b-11eb-8dff-9650c67e5759.png)
![Screen Shot 2021-03-02 at 9 44 27 AM](https://user-images.githubusercontent.com/21971636/109665184-e9c1a100-7b3b-11eb-884f-5e2de62f2245.png)

**Comment by mikriz** [comment]
Any look with this issue???
